lua/lspconfig/denols.lua

```diff
@@ -14,6 +82,20 @@ configs[server_name] = {
       unstable = false;
     };
   };
+  commands = {
+    DenolsCache = {
+      function()
+        buf_cache(0)
+      end;
+      description = "cache a module and all of its dependencies"
+    };
+    DenolsDefinition = {
```

Comment:
Why is this a command? Shouldn't this just transparently override the definition handler?

Reply:
It's ok. I'll fix it. Thank you.

Reply:
Just curious. I added the initial deno support, but I don't actively use it, so you're the expert :)